test/integration/query-interface.test.js

```diff
@@ -41,6 +41,28 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     });
   });
 
+  describe('showAllTables', () => {
+    it('should contain only tables', function() {
+      const cleanup = r => {
+        return this.sequelize.query('DROP VIEW V_Fail').then(() => r, () => r);
+      };
+      return this.queryInterface
+        .createTable('my_test_table', { name: DataTypes.STRING })
+        .then(cleanup)
```

Comment:
Change this to `.tap(cleanup)` to improve readability (and make my suggestions above work!)